sms/src/main/java/org/motechproject/sms/http/GenericResponseHandler.java

```diff
@@ -39,7 +38,7 @@ public void handle(OutgoingSms sms, String response, Header[] headers) {
                     sms.getRecipients().toString()));
             for (String recipient : sms.getRecipients()) {
                 getAuditRecords().add(new SmsRecord(getConfig().getName(), OUTBOUND, recipient, sms.getMessage(), now(),
-                        DeliveryStatus.DISPATCHED, null, sms.getMotechId(), providerMessageId, null));
+                        getTemplateOutgoingResponse().getSuccessStatus(), null, sms.getMotechId(), providerMessageId, null));
```

Comment:
Actually I think it is ok to use this status here, because in this this block we deal only with successfully send messages (it is checked by if instruction above) and in this place we put into database sms records only with DISPATCHED status.